docs/release-notes/1.0.0/release-1.0.0.md

```diff
@@ -0,0 +1,80 @@
+
+### Installation
+
+ML.NET supports Windows, MacOS, and Linux. See [supported OS versions of .NET Core 2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) for more details.
+
+You can install ML.NET NuGet from the CLI using:
+```
+dotnet add package Microsoft.ML
+```
+
+From package manager:
+```
+Install-Package Microsoft.ML
+```
+
+### Acknowledgements
+
+Shoutout to all these amazing contributors who have helped us with developing [ML.NET](https://aka.ms/mlnet)
+
+[amiteshenoy](https://github.com/amiteshenoy),
+[beneyal](https://github.com/beneyal),
+[bojanmisic](https://github.com/bojanmisic),
+[Caraul](https://github.com/Caraul),
+[dan-drews](https://github.com/dan-drews),
+[DAXaholic](https://github.com/DAXaholic),
+[dhilmathy](https://github.com/dhilmathy),
+[dzban2137](https://github.com/dzban2137),
+[elbruno](https://github.com/elbruno),
+[endintiers](https://github.com/endintiers),
+[f1x3d](https://github.com/f1x3d), 
+[feiyun0112](https://github.com/feiyun0112),
+[forki](https://github.com/forki), 
+[harshsaver](https://github.com/harshsaver),
+[helloguo](https://github.com/helloguo),
+[hvitved](https://github.com/hvitved),
+[Jongkeun](https://github.com/Jongkeun),
+[JorgeAndd](https://github.com/JorgeAndd),
+[JoshuaLight](https://github.com/JoshuaLight),
+[jwood803](https://github.com/jwood803),
+[kant2002](https://github.com/kant2002),
+[kilick](https://github.com/kilick),
+[Ky7m](https://github.com/Ky7m), 
+[llRandom](https://github.com/llRandom),
+[malik97160](https://github.com/malik97160),
+[MarcinJuraszek](https://github.com/MarcinJuraszek),
+[mareklinka](https://github.com/mareklinka), 
+[Matei13](https://github.com/Matei13),
+[mfaticaearnin](https://github.com/mfaticaearnin),
+[mnboos](https://github.com/mnboos),
+[nandaleite](https://github.com/nandaleite),
+[Nepomuceno](https://github.com/Nepomuceno)
+[nihitb06](https://github.com/nihitb06),
+[Niladri24dutta](https://github.com/Niladri24dutta),
+[PaulTFreedman](https://github.com/PaulTFreedman),
+[Pielgrin](https://github.com/Pielgrin),
+[pkulikov](https://github.com/pkulikov),
+[Potapy4](https://github.com/Potapy4),
+[Racing5372](https://github.com/Racing5372),
+[rantri](https://github.com/rantri), 
+[rantri](https://github.com/rantri), 
+[rauhs](https://github.com/rauhs),
+[robosek](https://github.com/robosek),
+[ross-p-smith](https://github.com/ross-p-smith),
+[SolyarA](https://github.com/SolyarA),
+[Sorrien](https://github.com/Sorrien), 
+[suhailsinghbains](https://github.com/suhailsinghbains),
+[terop](https://github.com/terop), 
+[ThePiranha](https://github.com/ThePiranha),
+[Thomas-S-B](https://github.com/Thomas-S-B),
+[timitoc](https://github.com/timitoc),
+[tincann](https://github.com/tincann),
+[v-tsymbalistyi](https://github.com/v-tsymbalistyi), 
+[van-tienhoang](https://github.com/van-tienhoang),
+[veikkoeeva](https://github.com/veikkoeeva),
+[yamachu](https://github.com/yamachu)
```

Comment:
>yamachu [](start = 1, length = 7)

should we add Zruty0 and galoshri?

Reply:
there are many MS folks who have worked on this at some point. We either add everyone or consider MS employees as part of team who worked on this.